packages/react-reconciler/src/__tests__/ReactBatchedMode-test.internal.js

```diff
@@ -53,7 +53,7 @@ describe('ReactBlockingMode', () => {
   }
 
   it('updates flush without yielding in the next event', () => {
-    const root = ReactNoop.createBlockingRoot();
```

Comment:
@gaearon this time I updated these tests instead of removing them. You were right they they're useful.

Reply:
Can probably rename this file to `ReactBatching-test` or something